tests/tests.yaml

```diff
@@ -95,12 +95,13 @@ answer_tests:
     - yt/frontends/owls/tests/test_outputs.py:test_snapshot_033
     - yt/frontends/owls/tests/test_outputs.py:test_OWLS_particlefilter
 
-  local_pw_042:  # PR 2504
+  local_pw_043:  # PR 3640
```

Comment:
@chrishavlin I think you forgot to bump the answer number again in your most recent commit

Reply:
oh! yes I did. thanks for catching!